### PR TITLE
Create addon.json

### PR DESCRIPTION
**Pull request by DerelictDrone**
Apparently we need one of these for the tool chain

### CODE REVIEW

**Comment by Anticept** [comment]
It's not for them to be allowed, its so that the toolchain can push to workshop.

File: addon.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+	"ignore"	:
+	[
+		"LICENSE",
+		"README.md",
+		"*.git",
+		"*.github",
+		"*.psd",
+		"*.vcproj",
+		"*.svn*"
+	]
```
Comment (Anticept) [inline addon.json:14]: [
		"*.psd",
		"*.vcproj",
		"*.txt",
		"*.xcf",
		".git*",
		"*.md",
		".glualint.json",
		".github",
		"git-hooks-pre-commit",
		"gitrid.sh",
		"LICENSE",
		"wiremod.*"
	]

File: addon.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+	"title"		:	"Wire CPU",
+	"type"		:	"ServerContent",
+	"tags"		:	[  ],
```
Comment (Anticept) [inline addon.json:4]: [ "build", "fun" ],

**Review by Anticept — CHANGES_REQUESTED** [review]
These are needed.

**Comment by DerelictDrone** [comment]
@Anticept Should I make it the "tool" type, same as the main wire addon uses?

**Comment by Anticept** [comment]
Yeah it should be a tool type honestly.

After that change, go ahead and self-merge and run a push to workshop so that the examples get uploaded.

**Pull request merged**